test/ssh_authorized_keys_test.rb

```diff
@@ -29,69 +31,78 @@
   let(:invalid_key_spec) { double("invalid_key") }
   let(:home) { "/home/user" }
   let(:key) { double("key") }
+  let(:keys) { [key] }
 
-  before do
-    allow(Yast::Users::SSHAuthorizedKey).to receive(:build_from)
-      .with(valid_key_spec)
-      .and_return(key)
-    allow(Yast::Users::SSHAuthorizedKey).to receive(:build_from)
-      .with(invalid_key_spec)
-      .and_return(nil)
-    ssh_authorized_keys.main
-  end
+  before { subject.main }
```

Comment:
Calling `main` explicitly is probably not needed anymore, it has been added [here](https://github.com/yast/yast-users/pull/113/files/4eca84cc0740f7cdbfd8ce1e8f80bd0e5905ed45..3d8a669442be7097662b58abdeab41cad0f4f6b8#diff-f2c8454f72a2b2ea8ee93bc0ab18fce1R97).